### PR TITLE
unwrap peth when exit

### PR DESCRIPTION
**Pull request by daiki44**
- WAK-110
- WAK-211

### CODE REVIEW

File: contracts/PlasmaETH.sol [hunk 1]
```diff
@@ -50,12 +29,32 @@ contract PlasmaETH is ERC20, ERC20Detailed {
...
+    }
+
+    /**
+     * @dev transfer PlasmaETH as ETH
+     */
+    function transfer(address _address, uint256 _amount) public returns (bool) {
+        require(
+            allowance(_address, msg.sender) >= _amount,
+            "PlasmaETH: transfer amount exceeds approved amount"
+        );
```
Comment (syuhei176) [inline contracts/PlasmaETH.sol:42]: question. I guess that once a user approve 10 ETH for Bob, the user can transfer 10 ETH many times over approved amount. Is this expected behavior in this PR?

Comment (daiki44) [reply contracts/PlasmaETH.sol:42]: @syuhei176 Thank you for your comment!
Probably I should add `decreaseAllowance()` to `transfer()`. What do you think about it?
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol#L188

Comment (daiki44) [reply contracts/PlasmaETH.sol]: updated!

File: contracts/PlasmaETH.sol [hunk 2]
```diff
@@ -50,12 +29,29 @@ contract PlasmaETH is ERC20, ERC20Detailed {
...
      */
     function unwrap(uint256 _amount) public {
+        _unwrap(msg.sender, _amount);
+    }
+
+    /**
+     * @dev transfer PlasmaETH as ETH
+     */
+    function transfer(address _address, uint256 _amount) public returns (bool) {
+        ERC20.transfer(_address, _amount);
```
Comment (syuhei176) [inline contracts/PlasmaETH.sol:39]: `require(ERC20.transfer(_address, _amount), "")` ?

File: contracts/PlasmaETH.sol [hunk 3]
```diff
@@ -50,12 +29,29 @@ contract PlasmaETH is ERC20, ERC20Detailed {
...
+    }
+
+    /**
+     * @dev transfer PlasmaETH as ETH
+     */
+    function transfer(address _address, uint256 _amount) public returns (bool) {
+        require(ERC20.transfer(_address, _amount), "failed ERC20.transfer");
+
+        // TODO: we can write `paybale(userAddress)` after v0.6.0
+        _unwrap(address(uint160(_address)), _amount);
```
Comment (syuhei176) [inline contracts/PlasmaETH.sol:42]: This transfer method was called by DepositContract and DepositContract can't burn owner's amount.

**Pull request merged**